logger/redact.go

```diff
@@ -74,10 +90,29 @@ func RedactJSON(json, replacement string, keys [][]string) string {
 				root.Set(k[0], r)
 			}
 		case l > 1:
-			if exist := root.Exists(k...); exist {
-				// get the parent obj then replace the value
-				v := root.Get(k[:l-1]...)
-				v.Set(k[l-1], r)
+			arrIndices := []int{}
+			for i, key := range k {
```

Comment:
what do you think about renaming `k` to `toRedact` and `r` to `replacement` for better readability on the variables used?

Reply:
Will rename.

Reply:
I renamed `r` to `replacementValue` instead since we have a `replacement` in the function parameter of `RedactJSON`. Since `redactArrayRecursive` takes in a `replacementValue *fastjson.Value`, I think it's OK to do that. What do you think? 8b54e87